Common/Orders/Fees/AlphaStreamsFeeModel.cs

```diff
@@ -0,0 +1,163 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using QLNet;
+using QuantConnect.Data.Fundamental;
+using QuantConnect.Orders.Fills;
+using QuantConnect.Securities;
+
+namespace QuantConnect.Orders.Fees
+{
+    public class AlphaStreamsFeeModel : FeeModel
+    {
+
+        private readonly decimal _forexCommissionRate = 0.000002m;
+        private decimal _liborRate;
+
+        private readonly Dictionary<string, EquityFee> _equityFee =
+            new Dictionary<string, EquityFee>();
+
+        private readonly Dictionary<string, CashAmount> _futureFee =
+            //                                                               Commission plus clearing fee
+            new Dictionary<string, CashAmount> { { Market.USA, new CashAmount(0.4m + 0.1m, "USD") } };
+
+        private readonly Dictionary<string, CashAmount> _optionFee =
```

Comment:
If the reason for the right aligned comment is so it appears near the numbers in `new CashAmount(...)`, perhaps formatting as this would help:

``` csharp
private readonly Dictionary<string, CashAmount> _optionFee = new Dictionary<string, CashAmount>
{
    // Commission plus clearing fee
    { Market.USA, new CashAmount(0.4m + 0.1m, "USD") }
};
```